index.css

```diff
@@ -701,3 +701,406 @@ body {
   margin-right: 20px;
 }
 
+@media screen and (min-width: 769px) {
```

Comment:
Your work section headline is also not displaying as per the Figma template. Consider fixing it.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/177909342-37adc113-dac2-411d-a638-da9412e32443.png) | ![image](https://user-images.githubusercontent.com/33268927/177909399-dc3ca4da-aadd-4f40-81e0-9cc37b971be0.png) |